ViewGenerator/tools/webpack/Rules/Files.ts

```diff
@@ -78,9 +44,9 @@ const getResourcesRuleSet = (assemblyName?: string, pluginsBase? : string): Rule
                                 }
 
                                 // URL (argument) is a relative path and contains the resource base path or the plugin assembly in its content
-                                if(isOutSystemsWebPackage) {
+                                if(isOutSystemsDesignSystemPackage) {
                                     let packg = url.substring(url.indexOf(PackagesFolder) + PackagesFolder.length);
-                                    return `/${pluginsBase}/node_modules/@outsystems/web${(isOutSystemsPackagesWithOldNaming(url)) ? "_" : "-"}${packg}`;
+                                    return `/${pluginsBase}/node_modules/@os-designsystem/${packg}`;
```

Comment:
we cant have this here... this is a general purpose public repo

Reply:
In the future, it can be... but for now it isn't. But I'm not seeing how we can make these agnostic...

Reply:
We can't have the rule to load all files on node_modules... I have to dig in a little more... Any insights?

Reply:
whats so special about this??? why cant this be like other packages????

Reply:
They can't because the assets are inside the component and references inside of it. Then the SS bundles everything again but these assets aren't bundled because the webpack doesn't have the loader prepared for that.

Reply:
I have an idea that I will try. I will add a new "configuration" on webpack-ouput-file on webviewimplementation.framework, there I will pass an array of paths from which assets should also be loaded.
That way the responsibility is from each project configuration and not on ReactView. 
What do you think @joaompneves

Reply:
@joaompneves After digging in the configurations, I will revert the code regarding the specific image treatment since this is not required I notice that the resources for components assets are being embedded on the ViewImplementation csproj